apps/tenant-management-webapp/src/app/pages/admin/services/script/editor/scriptEditorWrapper.tsx

```diff
@@ -171,18 +170,16 @@ export const ScriptEditorWrapper: FunctionComponent<ScriptEditorWrapperProps> =
   return (
     <section>
       <div>
-        {openAddScript ? (
-          <AddScriptModal
-            open={openAddScript}
-            isNew={true}
-            initialValue={selectedScript}
-            onCancel={() => {
-              reset();
-            }}
-            openEditorOnAdd={openEditorOnAdd}
-            onSave={saveScript}
-          />
-        ) : null}
+        <AddScriptModal
+          open={openAddScript}
+          isNew={true}
+          initialValue={selectedScript}
+          onCancel={() => {
+            reset();
+          }}
+          openEditorOnAdd={openEditorOnAdd}
+          onSave={saveScript}
+        />
```

Comment:
Make why was this change made? Is it for esthetics only? - make sure it still functions correctly - the original code would cause AddScriptModel to reset the data whenever it's opened - the new change may (depending on how the modal is written) cause some data to persist when it may not be wanted